<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawLineDialog</class>
 <widget class="QDialog" name="DrawLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Draw Line</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="point1GroupBox">
       <property name="title">
        <string>Point 1</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="x1FormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="x1Label">
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="x1SpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="y1FormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="y1Label">
            <property name="text">
             <string>y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="y1SpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="point2GroupBox">
       <property name="title">
        <string>Point 2</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QFormLayout" name="x2FormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="x2Label">
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="x2SpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="y2FormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="y2Label">
            <property name="text">
             <string>y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="y2SpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DrawLineDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DrawLineDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
